Makefile

```diff
@@ -219,7 +219,7 @@ ns1-secret:
 		--from-literal=apiKey=$${NS1_APIKEY}
 
 
-# run all linters from .golangci.yaml; see: https://golangci-lint.run/usage/install/#local-installation
+# runs golangci-lint aggregated linter
```

Comment:
Imho `.golangci.yaml` still looks useful in the comment

```suggestion
# runs golangci-lint aggregated linter; see .golangci.yaml for linter list
```

Reply:
Thanks, @ytsarev, makes sense, amended with suggested change.